tgui/packages/tgui/interfaces/PaperSheet.js

```diff
@@ -574,19 +570,16 @@ export const PaperSheet = (props, context) => {
   const {
     edit_mode,
     text,
-    paper_color,
+    paper_color = "white",
     pen_color = "black",
     pen_font = "Verdana",
     stamps,
     stamp_class,
-    stamped,
+    sizeX,
+    sizeY,
+    name,
   } = data;
-  // You might ask why?  Because Window/window content do wierd
-  // css stuff with white for some reason
-  const backgroundColor = paper_color && paper_color !== "white"
-    ? paper_color
-    : "#FFFFFF";
-  const stamp_list = !stamps || stamps === null
+  const stamp_list = !stamps
```

Comment:
Weird way of defaulting to an empty list.
```
  const stamp_list = stamps || [];
```
I'd also be fine with the below if you feel the ternary improves readability rather than relying on short-circuiting behavior.
```
  const stamp_list = stamps ? stamps : [];
```
(N.B. Can't easily do GitHub code suggestion here due to untouched lines below, suggestions above are meant to encompass the whole statement. Hopefully that was obvious, just clarifying.)